packages/core/strapi/lib/commands/opt-out-telemetry.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { resolve } = require('path');
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const fetch = require('node-fetch');
+const machineID = require('../utils/machine-id');
+
+const readPackageJSON = async path => {
+  try {
+    const packageObj = await fse.readJson(path);
+    const uuid = packageObj.strapi.uuid;
+
+    return [uuid, packageObj];
+  } catch (err) {
+    console.error(`${chalk.red('Error')}: ${err.message}`);
+  }
+};
+
+const writePackageJSON = async (path, file, spacing) => {
+  try {
+    await fse.writeJson(path, file, { spaces: spacing });
+    return true;
+  } catch (err) {
+    console.error(`${chalk.red('Error')}: ${err.message}`);
+  }
+};
+
+const sendEvent = async uuid => {
+  try {
+    await fetch('https://analytics.strapi.io/track', {
+      method: 'POST',
+      body: JSON.stringify({
+        event: 'didOptOutTelemetry',
+        uuid,
+        deviceId: machineID(),
+      }),
+      headers: { 'Content-Type': 'application/json' },
+    });
+  } catch (e) {
+    //...
+  }
+};
+
+module.exports = async function optOutTelemetry() {
+  const packageJSON = resolve(process.cwd(), 'package.json');
```

Comment:
```suggestion
  const packageJSONPath = resolve(process.cwd(), 'package.json');
```

Reply:
it's misleading to have packageJSON be a path and not the value itself

Reply:
Agree, this is misleading. Will change, thanks 👍